src/components/v5/shared/AvatarUser/AvatarUser.tsx

```diff
@@ -0,0 +1,67 @@
+import React, { FC } from 'react';
+import clsx from 'clsx';
+
+import Avatar from '~v5/shared/Avatar';
+import Link from '~v5/shared/Link';
+import { AvatarUserProps } from './types';
+
+const displayName = 'v5.AvatarUser';
+
+const AvatarUser: FC<AvatarUserProps> = ({
+  walletAddress,
+  avatar,
+  hasAvatarDecor,
+  userStatus,
+  userName,
+  size,
+  href,
+}) => {
+  const avatarImage = (
+    <span className="flex rounded-full items-center">
+      <span
+        className={clsx('flex rounded-full', {
+          'border-2 border-blue-400 ':
```

Comment:
border-2 is always added if there's avatar decor, so you should update the conditions to avoid code duplications.